bloomfilter.c

```diff
@@ -24,28 +26,28 @@ int modular_hash(const char string[], int seed_1, int seed_2)
  * @param bloomFilter: the filter to modify
  * @param string: the string to add to the bloom filter.
  */
-void add_string(bool bloom_filter[M], const char string[])
+void add_string(Bitarray* bloom_filter, const char string[])
```

Comment:
**question (non-blocking):** Should the parameter description mention that the arg is a pointer or is that redundant due to the header containing that information?

Reply:
> **question (non-blocking):** Should the parameter description mention that the arg is a pointer or is that redundant due to the header containing that information?

I would not mention that it is a pointer, I think that'd be like rewriting part of the code in documentation